readme.rst

```diff
@@ -305,7 +306,22 @@ get started, we will:
 Install prerequisite software
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To install the prerequisites:
+Before you start, you need to have ``make`` and ``python3`` on your system:
```

Comment:
i'm fine with this structural approach except i think we are being pedantic with the checks. can't we just do:

`sudo apt install python3 make`

and

`sudo apt install snapd nodejs`

btw, i personally did `apt install npm`. is `nodejs` better?